relay-server/src/services/processor/replay.rs

```diff
@@ -151,6 +161,8 @@ fn handle_replay_event_item(
             relay_log::warn!(
                 error = &error as &dyn Error,
                 ?event_id,
+                project_id = project_id.map(|v| v.to_string()),
```

Comment:
```suggestion
                project_id = project_id.map(|v| v.value()),
```

No need to allocate here.